app/views/publish/courses/a_level_requirements/what_a_level_is_required/new.html.erb

```diff
@@ -1,23 +1,48 @@
-<% content_for :page_title, title_with_error_prefix("SOME PAGE TITLE", @errors && @errors.any?) %>
+<% content_for :page_title, title_with_error_prefix(t("course.#{@wizard.current_step.model_name.i18n_key}.heading"), @wizard.current_step.errors && @wizard.current_step.errors.any?) %>
 
 <% content_for :before_content do %>
-  <%= govuk_back_link_to("change-me") %>
+  <%= govuk_back_link_to(
+      publish_provider_recruitment_cycle_course_a_levels_are_any_a_levels_required_for_this_course_path(
+        @provider.provider_code,
+        @provider.recruitment_cycle_year,
+        @course.course_code,
+        are_any_a_levels_required_for_this_course: { answer: "yes" }
```

Comment:
The next PR I am sorting out the params being passed from back and next steps. I wait for the third part because then I am having a deep thinking how to organise passing the params in each steps back and forward and then passing back to the instance of Wizard and the steps. So for now I just hardcoded the yes answer but this will be deleted/changed from this next PR.

Reply:
I don't have any particular ideas, but if you want to talk through your deep thinking, let me know, happy to chat it through before you tackle it.